content/getting-started-guides/gpio-explained.md

```diff
@@ -0,0 +1,189 @@
+# All what you've always wanted to know about GPIO
+
+This beginner document will explain what is GPIO and how to properly use it with concrete examples using .NET nanoFramework. You're in the perfect place if pull-up, pull-down seems like weird words for you. You're also in the good place if physical pin out and logical pin out needs a bit more clarity.
+
+## A bit of theory
+
+GPIO stands for "General Purpose Input/Output." In simple terms, [GPIO](https://en.wikipedia.org/wiki/General-purpose_input/output) refers to a set of pins on a microcontroller (MCU) or a computer that can be programmed to either input or output signals. These signals can be used to communicate with and control other devices or components in an embedded system.
+
+- General Purpose: These pins can be configured by the user or programmer for various tasks. They're not dedicated to one specific function but can be adapted for different purposes based on the needs of the system.
+- Input/Output: GPIO pins can either receive information (input) from external devices or send out information (output) to control external components. For instance, an input GPIO pin can detect whether a button is pressed or not, while an output GPIO pin can send a signal to turn on an LED or control a motor.
```

Comment:
If it's for beginners I would place something like:
"NOTE! Gpio pins are able to deliver low current. Connecting high demand devices may lead to destroy them"